ora2/jobs/SetOra2VersionOnSandbox.groovy

```diff
@@ -0,0 +1,59 @@
+/*
+    Variables consumed from the EXTRA_VARS input to your seed job in addition
+    to those listed in the seed job.
+
+    FOLDER_NAME: some-folder
+    NOTIFY_ON_FAILURE : e-mail-contact@example.com
+*/
+
+package ora2.jobs
+
+class SetOra2VersionOnSandbox {
+
+    public static def job = { dslFactory, extraVars ->
+        def jobRoot = extraVars.get("FOLDER_NAME")
+
+        def jobName = 'SetOra2VersionOnSandbox'
+        if (jobRoot != null && jobRoot != '') {
+            jobName = jobRoot + '/SetOra2VersionOnSandbox'
+        }
+
+        return dslFactory.job(jobName) {
+            parameters {
+
+                stringParam('ORA2_VERSION',
+                            'master',
+                            'The version/branch name of ORA2 to fetch/checkout on target machine'
+                            )
+                stringParam("SANDBOX_HOST",
+                            "staubina-ora2.sandbox.edx.org",
```

Comment:
If we're going to have a default, it should be not your sandbox, and should really get passed in from the seed job as an extraVar